src/plugins/imagery/components/ImageControls.vue

```diff
@@ -0,0 +1,272 @@
+                isPannable,
+                showCursorZoomIn,
+                showCursorZoomOut,
+                modifierKeyPressed
+            };
+        }
+    },
+    watch: {
+        imageUrl(newUrl, oldUrl) {
+            // reset image pan/zoom if newUrl only if not locked
+            if (newUrl && !this.panZoomLocked) {
+                this.$emit('resetImage');
+            }
+        },
+        cursorStates(states) {
+            this.$emit('cursorsUpdated', states);
+        }
+    },
+    mounted() {
+        document.addEventListener('keydown', this.handleKeyDown);
+        document.addEventListener('keyup', this.handleKeyUp);
+        this.clearWheelZoom = _.debounce(this.clearWheelZoom, 600);
+    },
+    beforeDestroy() {
+        document.removeEventListener('keydown', this.handleKeyDown);
+        document.removeEventListener('keyup', this.handleKeyUp);
+    },
+    methods: {
+        handleResetImage() {
+            this.$emit('resetImage');
+        },
+        handleUpdatePanZoom(options) {
+            this.$emit('panZoomUpdated', options);
+        },
+        toggleZoomLock() {
+            this.panZoomLocked = !this.panZoomLocked;
+        },
+        notifyFiltersChanged() {
+            this.$emit('filtersUpdated', this.filters);
+        },
+        handleResetFilters() {
+            this.filters = DEFAULT_FILTER_VALUES;
+            this.notifyFiltersChanged();
+        },
+        startDrag(e) {
+            e.preventDefault();
+            e.stopPropagation();
+        },
+        limitZoomRange(factor) {
+            return Math.min(Math.max(ZOOM_LIMITS_MIN_DEFAULT, factor), ZOOM_LIMITS_MAX_DEFAULT);
+        },
+        // used to increment the zoom without knowledge of current level
+        processZoom(increment, userCoordX, userCoordY) {
+            const newFactor = this.limitZoomRange(this.zoomFactor + increment);
+            this.zoomImage(newFactor, userCoordX, userCoordY);
+        },
+        zoomImage(newScaleFactor, screenClientX, screenClientY) {
+            if (!(newScaleFactor || Number.isInteger(newScaleFactor))) {
+                console.error('Scale factor provided is invalid');
+
+                return;
+            }
+
+            if (newScaleFactor > ZOOM_LIMITS_MAX_DEFAULT) {
+                newScaleFactor = ZOOM_LIMITS_MAX_DEFAULT;
+            }
+
+            if (newScaleFactor <= 0 || newScaleFactor <= ZOOM_LIMITS_MIN_DEFAULT) {
+                return this.handleResetImage();
+            }
+
+            this.handleUpdatePanZoom({
+                newScaleFactor,
+                screenClientX,
+                screenClientY
+            });
+        },
+        wheelZoom(e) {
+            // only use x,y coordinates on scrolling in
+            if (this.wheelZooming === false && e.deltaY > 0) {
+                this.wheelZooming = true;
+
+                // grab first x,y coordinates
+                this.processZoom(e.deltaY * 0.01, e.clientX, e.clientY);
```

Comment:
is 0.01 kinda `wheelZoomStep`?

Reply:
Added `ZOOM_WHEEL_SENSITIVITY_REDUCTION` constant, happy to shorten the name if there is something more concise.